x-pack/plugins/apm/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "Elastic",
+  "name": "apm",
+  "version": "8.0.0",
+  "private": true,
+  "license": "Elastic-License",
+  "scripts": {
+    "cypress:open": "../../../node_modules/.bin/cypress open --config fileServerFolder=new_e2e/cypress",
```

Comment:
I created a new `package.json` inside APM only to host the scripts needed to start the e2e tests. If that's not recommended let me know and I can figure something else.

Reply:
I don't think this is a good idea seeing the platform team went to great lengths to get a single unified package.json.

Instead we could have a `scripts` folder that simply contained these commands, and running them like:

`node apm/scripts/cypress`

Reply:
See this for instance https://docs.cypress.io/guides/guides/module-api.html#cypress-run